README.md

```diff
@@ -36,30 +39,23 @@ sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-12 120
 
 All of the required Python packages are installed in a Python Virtual Environment. The first step is to create the virtual environment and the second step is to load/activate the environment. The first step has to be run only once.
 
-Create the Python Virtual Environment:  
+Create the Python Virtual Environment on Ubuntu 22.04 LTS:  
 ```
 python3.11 -m venv .venv
 ```
 
-Load/activate the Python Virtual Environment: 
+Load/activate the Python Virtual Environment in Ubuntu 22.04 LTS: 
```

Comment:
We already mentioned that we only support Ubuntu 22.04 LTS Server Edition. Mentioning it again is not required.